site/docs/main/restic.md

```diff
@@ -1,40 +1,43 @@
 been waiting for a snapshot plugin for your storage platform, or if you're using EFS, AzureFile, NFS, emptyDir,
 local, or any other volume type that doesn't have a native snapshot concept, restic might be for you.
 
 Restic is not tied to a specific storage platform, which means that this integration also paves the way for future work to enable
-cross-volume-type data migrations. Stay tuned as this evolves!
+cross-volume-type data migrations.
 
-\* hostPath volumes are not supported, but the [new local volume type][4] is supported.
+**NOTE:** hostPath volumes are not supported, but the [local volume type][4] is supported.
 
-## Setup
+## Install restic
 
-### Prerequisites
+**Prerequisites**
+- Understand how Velero performs [backups with the restic integration](#how-backup-and-restore-work-with-restic).
+- [Download][3] the latest Velero release.
+- Kubernetes v1.10.0 and later. Velero's restic integration requires the Kubernetes [MountPropagation feature][6], which is enabled by default in Kubernetes v1.10.0 and later.
+- If you are using RancherOS, OpenShift, VMware Tanzu Kubernetes Grid Integrated Edition (formerly VMware Enterprise PKS), or Micrsoft Azure you must modify the restic DaemonSet spec before installing. See the [Configure restic DaemonSet spec](#configure-restic-daemonset-spec) section for more details.
 
-- Velero's restic integration requires the Kubernetes [MountPropagation feature][6], which is enabled by default in Kubernetes v1.10.0 and later.
+To install restic, use the `--use-restic` flag in the `velero install` command. See the [install overview][2] for more details on other flags for the install command.
 
-### Instructions
-
-Ensure you've [downloaded latest release][3].
+```
+velero install --use-restic
+```
 
-To install restic, use the `--use-restic` flag on the `velero install` command. See the [install overview][2] for more details. When using restic on a storage provider that doesn't currently have Velero support for snapshots, the `--use-volume-snapshots=false` flag prevents an unused `VolumeSnapshotLocation` from being created on installation.
+When using restic on a storage provider that doesn't currently have Velero support for snapshots, the `--use-volume-snapshots=false` flag prevents an unused `VolumeSnapshotLocation` from being created on installation.
 
-Please note: For some PaaS/CaaS platforms based on Kubernetes such as RancherOS, OpenShift and Enterprise PKS, some modifications are required to the restic DaemonSet spec.
+### Configure restic DaemonSet spec
+The steps in this section are only needed if you are installing on RancherOS, OpenShift, VMware Tanzu Kubernetes Grid Integrated Edition (formerly VMware Enterprise PKS), or Micrsoft Azure.
 
 **RancherOS**
 
-The host path for volumes is not `/var/lib/kubelet/pods`, rather it is `/opt/rke/var/lib/kubelet/pods`
+Update the host path for volumes from `/var/lib/kubelet/pods` to `/opt/rke/var/lib/kubelet/pods`.
```

Comment:
We should include that this change has to be made to the restic daemon set in the velero namespace.

Reply:
@ashish-amarnath, so to clarify, someone would need to run the velero install command and then do the steps in this section. I was talking to @nrb about this, and some of the information in this section a user should do before installing. Are their things in the `Configure restic DaemonSet spec` section that should be done before install?

Reply:
The restic daemon set is part of the installation when `--use-restic` is supplied to the `velero install command`
So prior to installation, there will not be a restic daemon set.

Reply:
@a-mccarthy Ashish is correct here and I was wrong when we spoke over Zoom. Updating the `hostPath` is done _after_ installation.

Reply:
@nrb and @ashish-amarnath i've update this back to the original so the deamonset configurations steps are after the velero install command